pkg/products/threescale/reconciler.go

```diff
@@ -413,7 +413,7 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, serverClient k8scl
 	}
 
 	// create the 3scale api manager
-	resourceRequirements := r.installation.Spec.Type != string(integreatlyv1alpha1.InstallationTypePDS)
+	resourceRequirements := r.installation.Spec.Type != string(integreatlyv1alpha1.InstallationTypeManaged)
```

Comment:
This should be `InstallationTypeWorkshop`:
```suggestion
	resourceRequirements := r.installation.Spec.Type != string(integreatlyv1alpha1.InstallationTypeWorkshop)
```